src/skai/beam_utils.py

```diff
@@ -15,7 +15,7 @@
 
 import pathlib
 import platform
-from typing import Tuple
+from typing import Tuple, Optional
```

Comment:
Do you still need Tuple and Optional here?

Reply:
These have been removed now